phpunit.xml.dist

```diff
@@ -8,6 +8,9 @@
   <filter>
     <whitelist>
       <directory suffix=".php">src</directory>
+      <exclude>
+        <directory suffix=".php">src/Logging/V2</directory>
```

Comment:
That seems like something we should look out for, yes.

`src/*/V?` and `src/*/V?beta?` then?

Should we plan for cases where a service reaches V10+?